packages/openapi-generator/src/parser/refs.ts

```diff
@@ -15,3 +19,81 @@ export function resolveObject<T>(
 ): T {
   return isReferenceObject(obj) ? refs.get(obj.$ref) : obj;
 }
+
+export async function createRefs(
```

Comment:
some docs for the methods

Reply:
I would move this as a static method of the class -> builder

Reply:
Done, but I kept an additional function for convenience as this much shorter and "functional" to write.